core/src/main/java/nl/nn/adapterframework/webcontrol/api/BrowseQueue.java

```diff
@@ -54,10 +56,14 @@
 	@Produces(MediaType.APPLICATION_JSON)
 	public Response getBrowseQueue() throws ApiException {
 		Map<String, Object> returnMap = new HashMap<String, Object>();
-
-		List<String> jmsRealms=JmsRealmFactory.getInstance().getRegisteredRealmNamesAsList();
-		if (jmsRealms.size()==0) jmsRealms.add("no realms defined");
-		returnMap.put("jmsRealms", jmsRealms);
+		JmsBrowser<javax.jms.Message> jmsBrowser = getIbisContext().createBeanAutowireByName(JmsBrowser.class);
+		Set<String> connectionFactories = new HashSet<String>();
```

Comment:
Could you use a LinkedHashSet instead of a plain HashSet? That keeps the order to the order of insertion, and therefor the ones used in the configuration on top of those 'only' in jsmRealms